packages/nimbus-icons/package.json

```
@@ -20,7 +33,7 @@
   },
   "scripts": {
     "build-icons": "pnpm svgr --filename-case kebab --typescript --jsx-runtime automatic --out-dir src/material-icons node_modules/@material-design-icons/svg/outlined",
-    "build": "pnpm run build-icons && tsc",
-    "dev": "tsc"
+    "build": "pnpm run build-icons && tsc --project tsconfig.esm.json & tsc --project tsconfig.cjs.json",
```

I managed to get the [build-process to work](https://github.com/commercetools/nimbus/pull/109/files) without committing the icons. The only "significant" change I did was running the two tsc tasks in sequence instead of in parallel/in the background:

```suggestion
    "build": "pnpm run build-icons && tsc --project tsconfig.esm.json && tsc --project tsconfig.cjs.json",
```